README.md

```diff
@@ -118,5 +118,5 @@ npm run publish remote=
 The Intern version badges were generated at https://shields.io with
 https://img.shields.io/badge/intern-v4-green.svg?colorB=2EC186.
 
-© [SitePen, Inc.](http://sitepen.com) and its
+© [OpenJS Foundation](https://openjsf.org/) and its
 [contributors](https://github.com/theintern/theintern.github.io/graphs/contributors)
```

Comment:
While we're fixing things... this should point to intern's contributor graph rather than theintern.github.io's.